public/src/components/PostItem/CommentsSection.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import CommentComponent from './CommentComponent';
+import methods from '../../utils/methods';
+
+const { sortComments } = methods;
+
+export default class CommentsSection extends React.Component {
+    constructor(props) {
+        super(props);
+        this.commentToShowStep = 20;
+        this.getComments = this.getComments.bind(this);
+        this.showMoreComments = this.showMoreComments.bind(this);
+        this.state = {
+            commentsToShow: this.commentToShowStep,
+        };
+    }
+
+    getComments() {
+        const { commentsToShow } = this.state;
+        const { comments } = this.props;
+        let sortedComments = sortComments([...comments]);
+        sortedComments = sortedComments
+            .filter(comment => comment.body !== 'null')
+            .slice(0, commentsToShow);
+
+        if (sortedComments) {
+            return sortedComments.map((comment, index) => (
+                <CommentComponent key={index}
+                                  {...comment}
+                />
+            ));
+        }
+        return null;
+    }
+
+    getLoadMoreButtonStyle() {
+        const { commentsToShow } = this.state;
+        const { comments } = this.props;
+        return {
+            display: comments.length - commentsToShow > 10 ? 'block' : 'none',
+        };
+    }
+
+    showMoreComments() {
+        const { commentsToShow } = this.state;
+        const { commentToShowStep } = this;
+        this.setState({
+            commentsToShow: +commentsToShow + +commentToShowStep,
+        });
+    }
+
+    render() {
+        return (
+            <div className="comments-section box">
+                {this.getComments()}
```

Comment:
I think all functions called directly from `render` method should be called like `renderComments`, `renderUserInfo`, etc
If you also group them together it will improve overall readability - all rendering work is at the bottom of component